app/main/views/services.py

```diff
@@ -157,6 +157,18 @@ def update_section(service_id, section):
                 "user",
                 "supplier app")
         except HTTPError as e:
+            errors_map = {}
+            for error in e.response.json()['error'].keys():
+                id = existing_service_content.get_question(error)['id']
```

Comment:
I get a `KeyError` on this line when `'errors'` contains non-field keys, e.g. `_form`